05-23 17:10:33 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:13:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:14:04 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:17:19 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:17:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:17:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:39:54 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "INSTRUCTOR" not found; SQL statement:
SELECT * FROM Instructor [42102-171]
05-23 17:42:18 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 17:45:17 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 17:47:12 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 17:51:37 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
05-23 17:56:54 jdbc[3]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:175)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:73)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1181)
	at org.apache.commons.dbcp.PoolingDataSource.getConnection(PoolingDataSource.java:106)
	at com.jajeem.util.ConnectionManager.getConnection(ConnectionManager.java:50)
	at com.jajeem.util.Connection.getConnection(Connection.java:19)
	at com.jajeem.util.BaseDAO.getConnection(BaseDAO.java:14)
	at com.jajeem.core.dao.h2.InstructorDAO.list(InstructorDAO.java:307)
	at com.jajeem.core.service.InstructorService.list(InstructorService.java:49)
	at com.jajeem.core.design.InstructorLogin.<init>(InstructorLogin.java:90)
	at com.jajeem.core.design.InstructorLogin.main(InstructorLogin.java:49)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-171]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 20 more
